<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(143, 240, 164);</string>
  </property>
  <widget class="QPushButton" name="b1_login">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
background-color: rgb(153, 193, 241);
font: 456 italic 18pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b2_exit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 10pt &quot;Ubuntu&quot;;
font: 456 italic 18pt &quot;Ubuntu&quot;;
background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(110, 160, 180);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(110, 160, 180);</string>
   </property>
  </widget>
  <widget class="QLabel" name="l2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 10pt &quot;Ubuntu&quot;;
font: 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLabel" name="l1">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_data">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 11pt &quot;Ubuntu&quot;;
border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 456 italic 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>       welcome</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
